graphql_compiler/cost_estimation/analysis.py

```diff
@@ -56,6 +63,21 @@ def _convert_int_interval_to_field_value_interval(
     return Interval(lower_bound, upper_bound)
 
 
+def _get_location_vertex_path(location: BaseLocation) -> VertexPath:
+    """Get the VertexPath for a BaseLocation pointing at a vertex."""
+    if location.field is not None:
+        raise AssertionError(f"Location {location} represents a field.")
```

Comment:
nit: This error is factually correct but doesn't actually describe the precondition that was violated.